test/e2e/hazelcast_wan_test.go

```diff
@@ -318,8 +321,8 @@ var _ = Describe("Hazelcast WAN", Label("hz_wan"), func() {
 		waitForMapSizePortForward(context.Background(), hzCrs[hzTarget1], localPort, map21, currentSize, 1*Minute)
 	})
 
-	//When("Map is given twice in the WAN spec",
-	It("should continue replication if one reference is deleted ", Label("slow"), func() {
+	//When("Map is given twice in the Wan spec",
```

Comment:
```suggestion
	//When("Map is given twice in the WAN spec",
```

I think WAN is more correct usage 🙂